src/measure_theory/interval_integral.lean

```diff
@@ -389,63 +389,107 @@ begin
   { simp [(show 0 < c, from h), mul_div_cancel, hc, abs_of_pos] }
 end
 
+@[simp] lemma integral_comp_mul_right' (c) :
+  c • ∫ x in a..b, f (x * c) = ∫ x in a*c..b*c, f x :=
```

Comment:
Why smul and not `mul` here?

Reply:
Because `c` is a member of `ℝ` and the integral is a member of `E`.